docs/app/pages/components/components-sections/page-header/page-header/page-header.component.html

```diff
@@ -47,9 +47,12 @@
 </uxd-api-properties>
 
 <uxd-api-properties tableTitle="Outputs">
-    <tr uxd-api-property name="backClick" type="EventEmitter">
+    <tr uxd-api-property name="backClick" type="MouseEvent">
         The event fired when the back button is clicked.
     </tr>
+    <tr uxd-api-property name="logoClick" type="MouseEvent">
+        The event fired when the image at the top left part of the header is clicked.
```

Comment:
Actually, could be remove this `logoClick` documentation here, I'm just thinking that the logo never appears in this theme. Instead we should create a separate PR to document this in the MF documentation only

Reply:
Removed and will add to micro focus site documentation